sansa-owl-spark/src/main/scala/net/sansa_stack/owl/spark/ExtractOWLAxioms.scala

```diff
@@ -0,0 +1,168 @@
+package net.sansa_stack.owl.spark
+
+import scala.collection.JavaConverters._
+
+import org.apache.spark.rdd.RDD
+import org.semanticweb.owlapi.model._
+
+package object extractOWLAxioms {
+
+
+  def Declarations (axioms: RDD[OWLAxiom]): RDD[OWLDeclarationAxiom] =
```

Comment:
Just as a note: I'd prefer method names that follow the Scala naming conventions, and in this case the method name looks like a plain accessor but does some actual computation which is (at least to me) confusing as well. This holds for all methods of the `extractOWLAxioms` object. Moreover in terms of the "Don't repeat yourself" principle having one method which takes the axiom type as one arguments seemed more appropriate here.